src/riscv/Makefile

```diff
@@ -5,7 +5,7 @@ OBJCOPY = $(CROSS)objcopy
 SIZE = $(CROSS)size
 
 # RISC-V specific options
-CFLAGS=-Wall -Ofast -march=rv32im -mabi=ilp32 -ffreestanding -flto -fomit-frame-pointer -Wl,--gc-section --specs=nano.specs -I..
+CFLAGS=-Wall -Ofast -march=rv32imafc -mabi=ilp32f -ffreestanding -flto -fomit-frame-pointer -Wl,--gc-section --specs=nano.specs -I..
```

Comment:
Don't change `CFLAGS`. The doom should rely on RV32IM for compatibility considerations.